jupyterlab_automl/src/components/list_resources_panel.tsx

```diff
@@ -334,12 +428,20 @@ export class ListResourcesPanel extends React.Component<Props, State> {
   }
 
   private async getDatasets() {
-    const datasets = await DatasetService.listDatasets();
-    this.setState({ datasets: datasets });
+    try {
+      const datasets = await DatasetService.listDatasets();
+      this.setState({ datasets: datasets });
+    } catch (e) {
+      console.log('Failed to load dataset resource.');
```

Comment:
This error would only show if the user didn't have the API installed. But now that we've added checking to make sure the API is enabled we can probably take this out because the user would never get here anyways.

Reply:
I think you could leave it in, but maybe add a comment on why the user doesn't need to be alerted. I like to see comments used sparingly, but they can be great for explaining *why* you're doing something.